src/histolab/slide.py

```diff
@@ -411,13 +605,37 @@ def _thumbnail_size(self) -> Tuple[int, int]:
         Tuple[int, int]
             Thumbnail size
         """
+        if self._use_largeimage:
+            raise MayNeedLargeImageError(
+                "When use_largeimage is set to True, the thumbnail is fetched "
+                "by the large_image module. Please use thumbnail.size instead."
```

Comment:
yes this will never happen

Reply:
@alessiamarcolini @ernestoarbitrio Sorry for the late response. Actually this does happen and it happened for a couple of cases I worked with. The way `large_image` fetches the thumbnail is different depending on the tile source. If it's a slide compatible with openslide then yes it will most likely be the same, but this will not always be the case. Hence deferring to `large_image` for calculating the size for us.